editor/src/uuiui/widgets/layout/ui-row.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled'
+import { commonSenseUtopiaLayoutShorthands } from './common-layout-shorthands'
+import { flexRowStyle } from '../../styles/layout-styles'
+import * as React from 'react'
+import { UtopiaTheme } from '../../styles/theme'
+
+export interface UIRowProp extends React.InputHTMLAttributes<HTMLDivElement> {
+  padded?: boolean
+  rowHeight?: keyof typeof UtopiaTheme.layout.rowHeight
+}
+
+export const UIRow = styled.div<UIRowProp>((props) => ({
+  ...commonSenseUtopiaLayoutShorthands,
+  ...flexRowStyle,
+  padding: props.padded ? UtopiaTheme.layout.rowHorizontalPadding : undefined,
+  height: props.rowHeight ?? UtopiaTheme.layout.rowHeight.normal,
```

Comment:
This looks wrong. Shouldn't it be something like:
```
height: 
  props.rowHeight == null ? 
    UtopiaTheme.layout.rowHeight.normal :
    UtopiaTheme.layout.rowHeight[props.rowHeight]
```
since `props.rowHeight` will be a string (e.g. `'tall'`)?